lib/utils.js

```diff
@@ -0,0 +1,25 @@
+var path = require('path');
+
+function isFileURL(url) {
+  return url.substr(0, 5) === 'file:';
+}
+exports.isFileURL = isFileURL;
+
+/* Remove scheme prefix from file URLs, so that they are paths. */
+function filePath(url) {
+  if (isFileURL(url))
+    return url.replace(/^file:\/+/, '/');
+}
+exports.filePath = filePath;
+
+/* Coerce URLs to paths, assuming they are file URLs */
+function coercePath(url) {
+  // give absolute URLs benefit of the doubt that they are file paths
+  if (path.isAbsolute(url))
```

Comment:
Would argue that it makes as much sense as relating relative paths to process directory - it's just consistency. We actually get this behaviour for-free from that case though (at the cost of calling `process.cwd()` despite its value having no effect on the result) - so happy to remove this branch.